ember-highcharts/src/components/high-charts.ts

```diff
@@ -202,81 +203,66 @@ export default class HighCharts<
    * Dynamically imports the necessary pieces from Highcharts, based on chart type and options.
    */
   async _importHighchartsDeps() {
+    let tmpModule;
+
     if (this.args.mode === 'Map') {
-      Highcharts = await waitForPromise(import('highcharts/highmaps'));
```

Comment:
We don't need highmaps for maps anymore?

Reply:
Well we do, but now it's a module: `import('highcharts/modules/map')` or `import('highcharts/modules/stock')`. Just pushed an update. My understanding is that:

1) You always need to `import('highcharts')`
2) And then depending on what "extras" you want you pull in any of the modules